librz/include/rz_vector.h

```diff
@@ -243,6 +255,18 @@ static inline void **rz_pvector_data(RzPVector *vec) {
 	return (void **)vec->v.a;
 }
 
+// returns a pointer to the first element of the vector
+static inline void **rz_pvector_head(RzPVector *vec) {
```

Comment:
This contradicts what the function actually does, which would be
```c
// returns the first element of the vector
static inline void *rz_pvector_head(RzPVector *vec) {
```
(notice also the return type change)